shared/common-adapters/avatar.render.desktop.js

```diff
@@ -56,19 +56,13 @@ const BackgroundDiv = glamorous.div(
     top: backgroundOffset,
   },
   props => ({
-    backgroundColor: props.loaded ? globalColors.white : props.loadingColor || globalColors.lightGrey,
+    backgroundColor: props.loadingColor || globalColors.lightGrey,
     borderRadius: props.borderRadius,
   })
 )
-class Background extends React.PureComponent<{loaded: boolean, loadingColor: ?string, borderRadius: any}> {
+class Background extends React.PureComponent<{borderRadius: any}> {
   render() {
-    return (
-      <BackgroundDiv
-        loaded={this.props.loaded}
-        loadingColor={this.props.loadingColor}
-        borderRadius={this.props.borderRadius}
-      />
-    )
+    return <BackgroundDiv borderRadius={this.props.borderRadius} />
```

Comment:
Do you still want to pass `loadingColor` here?

Reply:
i don't think this is ever set